blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -100,25 +105,28 @@ class OrganizationPipelines extends Component {
             delete this.jobListener;
         }
     }
-
-    _selectPipeline(pipelines, params) {
-        if (pipelines && params) {
-            const { pipeline } = params;
-            const p = pipelines.find(aPipeline => aPipeline.fullName === pipeline);
-            if (!p) {
-                return new Failure(`${pipeline} not found`);
-            }
-            return p;
+    
+    _getOrganizationName(nextProps) {
+        if (nextProps && nextProps.params) {
+            return nextProps.params.organization;
+        }
+        if (this.props && this.props.params && this.props.params.organization) {
+            return this.props.params.organization;
+        }
+        if (this.context && this.context.params && this.context.params.organization) {
+            return this.context.params.organization;
```

Comment:
I'm sure it's naivety wrt react on my behalf, but the fact that you needed to write code like the above highlights to me why react props/context is a shity way of passing data around. 😢